scss/reset/_customProperties.scss

```diff
@@ -0,0 +1,44 @@
+//
```

Comment:
Let's assume we'll be moving more Sass vars into custom properties to create interoperability with non-Sass styling in consumer apps.

We could create a directory `scss/utils/customProperties/` for different types of custom properties to be defined.

This `_customProperties.scss` file could then move to `scss/utils/customProperties/_responsive.scss`